Docs: Document that opacity uses layers (#2600)
slint-ui/slint: docs/language/src/builtins/elements.md

-   **`cache-rendering-hint`** (_in_ _bool_): When set to `true`, this provides a hint to the renderer to cache the contents of the element and all the children into an intermediate cached layer. For complex sub-trees that rarely change this may speed up the rendering, at the expense of increased memory consumption. Not all rendering backends support this, so this is merely a hint. (default value: `false`)
-   **`dialog-button-role`** (_in_ _enum [`DialogButtonRole`](enums.md#dialogbuttonrole)_): Specify that this is a button in a `Dialog`.
-   **`opacity`** (_in_ _float_): A value between 0 and 1 (or a percentage) that is used to draw
    the element and its children with transparency.
    0 is fully transparent (invisible), and 1 is fully opaque.

>     If the element has children, the opacity is applied to the whole tree of children as if they
>     were first drawn into an intermediate layer, and the whole layer is rendered with a different opacity.
>     (default value: 1)

Reviewer: ```suggestion
    The opacity is applied to the tree of child elements as if they
    were first drawn into an intermediate layer, and then the whole layer is rendered with this opacity.
    (default value: 1)
```
